### DISCUSSION TITLE
How to group together form fields in different section?

### DISCUSSION THREAD

**Discussion by pradeep-pandit** [serenity-is/Serenity#6121] [Help] [answered]
I want to show my form fields as different sections such as Personal details, Contact details, Identification details just like new customer form in Northwind customer.
![image](https://user-images.githubusercontent.com/94299697/143764682-d1a8625c-a653-4725-a624-910c70c41f9b.png)

**Comment by edwardch** [comment, marked as answer]
Check the code for that page [here](https://github.com/serenity-is/common-features/blob/be0698d54ae22616f04a3d818c053e274a186423/src/Serenity.Demo.Northwind/Serenity.Demo.Northwind/Customer/CustomerForm.cs#L12)

In YourForm.cs, add 
```cs
[Category("General")]
// CustomerID
// CompanyName
[Category("Contact")]
// ContactName
// ContactTitle
// etc
```

**Reply by pradeep-pandit** [reply]
Thanks @edwardch for you help.
